src/config.ts

```diff
@@ -14,11 +20,26 @@ export type Config = {
 const defaultConfig: Config = {
   asyncUtilTimeout: 1000,
   defaultHidden: true,
+  defaultIncludeHiddenElements: true,
 };
 
 let config = { ...defaultConfig };
 
 export function configure(options: Partial<Config>) {
+  if (
```

Comment:
I wonder if we should rather than syncing these two keys have a solution that would set `defaultIncludeHiddenElements` when user passes `defaultHidden`:

```ts
const { defaultHidden, ...restOptions } = options;
const optionsToSet {
  ...restOptions,
  defaultIncludeHiddenElements: restOptions.defaultIncludeHiddenElements ?? defaultHidden ?? config.defaultIncludeHiddenElements
}
```

That way we would also simplify the consuming code. 

Regarding typing this could be handled as:
```ts
interface Config {
  asyncUtilTimeout: number;
  defaultIncludeHiddenElements: boolean;
  // no defaultHidden
}
let config: Config

interface ConfigAliasOptions = {
  defaultHidden?: boolean;
}

function configure(config: Config & ConfigAliasOptions)
```

@MattAgn wdyt?

Reply:
yes I like that!